cfn-resources/go.mod

```diff
@@ -10,9 +10,9 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/cloudformation v1.47.0
```

Comment:
Is it possible that this PR is missing the adjustments to the `templateMakeFile`? Would make sure to also include the `create-test-resources` `delete-test-resources` and `run-contract-testing` make commands as they are present for all resources that run with contract tests in CI.